.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
 # Default owners
-*       @hashicorp/hds-engineering
\ No newline at end of file
+*       @hashicorp/hds-engineering
+
+# co-ownership of website docs & assets
+/website/docs/ /website/public/assets/ @hashicorp/hds-engineering @hashicorp/hds-design
```

Comment:
issue: I think docs and assets need to be done on separate lines, github says its an invalid owner
![Screenshot 2024-11-19 at 10 26 07 AM](https://github.com/user-attachments/assets/4664d112-0663-4c94-9402-3b69c3ee7f72)

Reply:
🤦

Reply:
I was about to leave the same comment – it's a shame you can't chain them